lib/oli_web/live/collaboration_live/collab_space_view.ex

```diff
@@ -532,7 +532,4 @@ defmodule OliWeb.CollaborationLive.CollabSpaceView do
 
   defp is_archived?(:archived), do: true
   defp is_archived?(_), do: false
-
-  defp send_updated_posts(:submitted, _), do: send(self(), :updated_posts)
```

Comment:
Here, it's necessary broadcast the post that is sent always, since we may have instructors to broadcast that post in case approval is required. We have no way of differentiate when a post is broadcasted to a student or to an instructor. 
Are we okay with this?